app/jobs/characterize_job.rb

```diff
@@ -2,9 +2,10 @@ class CharacterizeJob < ActiveJob::Base
   queue_as CurationConcerns.config.ingest_queue_name
 
   # @param [FileSet] file_set
-  # @param [String] filename a local path for the file to characterize. By using this, we don't have to pull a copy out of fedora.
+  # @param [String] filename a local path for the file to characterize so we don't have to pull a copy out of fedora.
   def perform(file_set, filename)
-    Hydra::Works::CharacterizationService.run(file_set, filename)
+    return unless file_set.characterization_proxy.content
```

Comment:
@mjgiarlo I don't really know, honestly, but if there's no content then things would crash in weird ways. Perhaps we should raise an informative exception message?